main.js

```diff
@@ -0,0 +1,63 @@
+var ReReddit = {
+  newPost: function () {
+    var postTemplate = "<div class=\'post\'><div class=\"post-text-container\"><h4 class=\"post-text\">" + $('.text-input').val() + "</h4></div><h4> - " + "Posted by: <strong>" + $('.name-input').val() + "</strong></h4>" +"<p><button class=\'remove btn-danger btn-xs\'>remove</button> <button class=\'comment btn-xs btn-warning\'>comment</button><button class=\"edit-button btn-xs btn-success\">edit</button></p>";
+    
+    var commentTemplate = "<div class=\"all-comments hidden\"><div class=\"comment-container\"></div><div class=\"comment-input\"><div class=\"form-group\">" + 
+    "<input type=\"text\" placeholder=\"Comment\" class=\"comment-text form-control\"/>" + "</div>" + "<div class=\"form-group\">" + "<input type=\"text\" placeholder=\"Name\" class=\"comment-name form-control\"/>" + "</div>" + "<div class=\"clearfix\">" + "<button class=\"submit-comment-button btn btn-info btn-xs\">Add Comment</button>" + "</div>" + "</div></div></div>";
+
+    $('.blog-container').append(postTemplate + commentTemplate);
+
+    $('.remove').unbind().click((e) => {
```

Comment:
Why `unbind`? I think it was deprecated anyway in favor of `off` (the opposite of `on`)